### DISCUSSION TITLE
optimum refreshrate for displays

### DISCUSSION THREAD

**Discussion by andrejetcat** [flybywiresim/aircraft#4792] [Q&A]
I think, I read some posts about the cockpit display refreshrate to make the whole experience more smooth. Is it right that it is recommended to set it on "high" to get better results compared to "medium"? I can´t test it at the moment. anybody know  it?

**Comment by solarisfire** [comment]
This totally depends on the specifications of your system.

Turning things up will only make things more smooth if your system has the overhead to draw the extra frames needed.

I'm not sure there's a setting that would be best for everyone.

**Reply by andrejetcat** [reply]
okay thank you for your answer. I remember read something that stated an "optimum" setting for the displays. Thank you !

**Reply by CptBayer** [reply]
my experience is right now a setting of high instead of medium. Due to an early development state of the fbw mod, medium was better for my system.. this changed and I get smoother results with high ..... 

greets Klaus
